pkg/images/utils/utils.go

```diff
@@ -236,3 +236,25 @@ func FilterSuppressedCVEsNoClone(img *storage.Image) {
 		}
 	}
 }
+
+// UniqueImageNames returns the unique image names from the two given slices of image names.
+func UniqueImageNames(a, b []*storage.ImageName) []*storage.ImageName {
+	uniqueImageFullNames := set.NewStringSet()
```

Comment:
I think you can make a `set.NewSet[*storage.ImageName]()` and then call `.AsSlice()`

Reply:
It won't work, since the set will use the equal method of the pointer, but not the proto (i.e. it won't use `proto.Equal` to check whether both resources are equal or not).

However, I could also just use `protoutils.SliceContains` instead which I did now.